__tests__/app.test.js

```diff
@@ -82,4 +82,66 @@ describe('/api/articles/:id', () => {
             expect(err.response.body.msg).toEqual("Article does not exist")
         })
     })
+})
+describe('/api/articles', () => {
+    test('responds with an array of article objects with comment_count created from comments.js', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles
+            expect(articles).toBeInstanceOf(Array);
```

Comment:
This expect statement is redundant as you're using an array method lower down that would fail if it wasn't an array 😄